css/style.css

```diff
@@ -15,5 +14,234 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+header,
+main,
+
+footer {
+    padding: 0 1.5rem;
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+header {
+    margin-top: 30px;
```

Comment:
We should be OK without this - it makes it so that there's quite a bit more whitespace on top of the nav than the picture asks for